### DISCUSSION TITLE
Sharing Triggers

### DISCUSSION THREAD

**Discussion by simmotech** [quartznet/quartznet#1300] [General]
From the Getting Started documentation...

> Why Jobs AND Triggers? Many job schedulers do not have separate notions of jobs and triggers.

Why can't a single Trigger be shared by multiple Jobs?

**Comment by lahma** [comment]
Because it's the other way around, job can be shared by triggers 😉 

Usually you have less jobs than triggers, a common scenario could be that you have `NotificationJob` (say it sends latest updates or whatnot) and then you can have as many triggers as you need, even one for each user. So the triggers are parametrized to have the user id in their job data map, this way you can use same job to send notifications and provide the target user id and the user's chosen schedule (only Mondays for example).

**Comment by simmotech** [comment]
Your example makes sense of course. 
My scenario is somewhat different in that the Jobs already contain the information they need so it that's how it works then I'll just create trigger(s) per job.
